artifacts/definitions/Server/Utils/CreateCollector.yaml

```diff
@@ -149,12 +154,29 @@ parameters:
 
       LET _ <= log(message="Will collect package " + filename)
       LET report_filename <= if(condition=Template, then=filename + ".html")
-      SELECT * FROM collect(artifacts=Artifacts, report=report_filename,
-          args=Parameters, output=filename + ".zip", template=Template,
-          cpu_limit=CpuLimit,
-          progress_timeout=ProgressTimeout,
-          timeout=Timeout,
-          password=Password, level=Level, format=Format)
+      LET X <=  SELECT format(format="%02x", args=rand(range=255)) AS A FROM range(end=25) 
+      LET RandPass <= join(array=X.A)          
+
+      SELECT * FROM switch(
+        a={SELECT * FROM collect(artifacts=Artifacts, report=report_filename,
```

Comment:
I wonder if we can collect these into a single case by just adding metadata if needed?

Reply:
You mean something like `SELECT * FROM collect(...,metadata=metadata | Null)` ?